cmd/cloud.go

```diff
@@ -33,6 +33,7 @@ type cmdCloud struct {
 	uploadOnly    bool
 }
 
+//nolint:dupl // remove this statement once the migration from the `k6 cloud` to the `k6 cloud run` is complete.
```

Comment:
The _why_ is missing. I don't get why we have this nolint rule, it requires me to check the code in detail.